container.go

```diff
@@ -496,11 +496,13 @@ func (c *Container) fetchState(cmd string) (State, error) {
 
 func (c *Container) getSystemMountInfo() {
 	// check if /dev needs to be bind mounted from host /dev
+	c.systemMountsInfo.BindMountDev = false
+
 	for _, m := range c.mounts {
-		if m.Source == "/dev" && m.Destination == "/dev" && m.Type == "bind" {
-			c.systemMountsInfo.BindMountDev = true
-		} else {
-			c.systemMountsInfo.BindMountDev = false
+		if m.Destination == "/dev" {
```

Comment:
Nice catch - in the original case, it would only return true of the last of the mounts happened to be the bind mount from host /dev?  

Any rationale for dividing it into two if clauses?

Reply:
@amshinde the logic of the patch works, but I have the same question than @egernst, why don't we keep only one if statement ?
```go
if m.Source == "/dev" && m.Destination == "/dev" && m.Type == "bind" {
```

Reply:
Also @amshinde, can we create a test to assert the expected behaviour (here, in `runtime` or `tests`)?

Reply:
@sboeuf I just add the split to make it clear we need to care of the destination first, thats all. But yeah we could very well have it as one statement.